Assets/Project/Scripts/Networks/NetworkService.cs

```diff
@@ -5,15 +5,15 @@ namespace Project.Scripts.Networks
 {
     public static class NetworkService
     {
-        #if UNITY_EDITOR
+        #if DEVELOPE
         // ローカルデバッグ用サーバーIP
         private static readonly string _serverIp = "localhost";
         #else
         // TODO 本番サーバーIP
         private static readonly string _serverIp = "";
         #endif
 
-        #if UNITY_EDITOR
+        #if DEVELOPE
```

Comment:
```suggestion
        #if DEVELOP
```
ありとあらゆるところが develope になっている・・・

Reply:
あ，`PRODUCTION` なら `DEVELOPMENT` かも
他のパターンとしては，`PROD` と `DEV` とか

Reply:
`PRODUCT` と `DEVELOP` にする

Reply:
PRODUCTION，DEVELOPMENTの方がしっくりくる

Reply:
`PRODUCTION` の方がわからないけど、 `DEVELOP` の方がよく使われてると思うけど

Reply:
PROD，DEVとかも多い．
俺の経験だと，dev か development が多い．develop っていうのが変なのかもしれないけど

Reply:
もしくは `DEBUG` / `RELEASE`

Reply:
dev prod じゃだめ？

Reply:
じゃあ `DEV` `PROD` で

Reply:
オッケ